### PR TITLE
Some little Improvements

### PR DESCRIPTION
**Pull request by louis49**
Tensorboard summary improvements (date, title, learning rate, step) + shell script
Warning fix
Fix to use anchor as train method
Fix on BiLSTM class

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 import app.ops as ops
 import app.ozers as ozers
 import app.utils as utils
-
+import datetime as datetime
```
Comment (khaotik) [inline main.py]: It's a convention to put built-in imports above local module imports.

File: main.py [hunk 2]
```diff
@@ -396,13 +400,16 @@ def build(self):
 
     def train(self, n_epoch, dataset):
         global g_args
-        train_writer = tf.summary.FileWriter(hparams.SUMMARY_DIR, g_sess.graph)
+        train_writer = tf.summary.FileWriter(hparams.SUMMARY_DIR + '/' + str(datetime.datetime.now().strftime("%m%d_%H%M%S")) + ' ' + hparams.SUMMARY_TITLE, g_sess.graph)
```
Comment (khaotik) [inline main.py]: Even it doesn't support windows right now, it's better to use `os.path.join` here.

File: main.py [hunk 3]
```diff
@@ -501,7 +511,7 @@ def train(self, n_epoch, dataset):
     def test(self, dataset, subset='test', name='Test'):
         global g_args
         train_writer = tf.summary.FileWriter(
-            hparams.SUMMARY_DIR, g_sess.graph)
+            hparams.SUMMARY_DIR + '/' + str(datetime.datetime.now().strftime("%m%d_%H%M%S")) + ' ' + hparams.SUMMARY_TITLE, g_sess.graph)
```
Comment (khaotik) [inline main.py]: idem

File: tensorboard.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+open -a 'Google Chrome' http://localhost:6008/#scalars
```
Comment (khaotik) [inline tensorboard.sh]: This command doesn't work on my OS (Ubuntu 16.04). Also I don't think everyone uses Chrome.

**Review by khaotik — CHANGES_REQUESTED** [review]
Thanks for the PR! I have a few small comments.

**Comment by louis49** [comment]
Done

**Pull request merged**